core/templates/pages/signup-page/signup-page.component.html

```diff
@@ -119,7 +119,7 @@
           <input type="checkbox"
                  [(ngModel)]="hasAgreedToLatestTerms"
                  [ngModelOptions]="{standalone: true}"
-                 class="protractor-test-agree-to-terms-checkbox"
+                 class="protractor-test-agree-to-terms-checkbox oppia-checkbox-margin"
```

Comment:
Lets move the `oppia-checkbox-margin before the other class.`

Reply:
can i shift this class to line 142 in this file ?

Reply:
Not sure what you mean, just do:
```suggestion
                 class="oppia-checkbox-margin protractor-test-agree-to-terms-checkbox"
```

Reply:
Ok sir